### ISSUE TITLE
What Skills Will Volunteers Need?

### ISSUE THREAD

**Issue by Emersont1** [Emersont1/TechNottingham.IT-Refurb#8]
c.se:party-wizard:  29 minutes ago
Not everyone involved has to have hardware skills and we shouldn't make that a requirement. Dealing with machines that just need to be reset and have software installed can be done by people who have a different skill set to people who can do PC repair

**Comment by cruxicheiros** [comment]
I think we can start by listing skills which individuals might have which would be useful. Not everyone has to be able to do more than one thing, and allowing people to pick roles based on what they are already good at would be efficient.

This is what I can think of off the top of my head

- Coordinating volunteer efforts
- Managing logistics [for shipping computers, etc]
- Doing repairs on or assessing desktop PCs
- Doing repairs on or assessing laptops
- Wiping hard disks / installing new OSes / resetting computers / installing software

**Comment by cruxicheiros** [comment]
**Following Slack discussion**

The main process can really be divided into three distinct parts:

**Step 1:** Work out which volunteers should get what devices.
**Step 2:** Have volunteers reset and install software on each device.
**Step 2b:** If the device isn't suitable, have a different stream of volunteers ready to take its parts/fix it/send for recycling
**Step 3:** Do QA on the reset devices

The QA stage is to to ensure that they meet quality standards, make sure no volunteers have done anything nefarious to the devices (I personally think this is unlikely but it needs to be part of it) and we know which hardware works and doesn't work before issuing devices.

These stages should ideally be performed by three separate groups of people: someone checking devices to send to volunteers has a totally different workflow to someone resetting and installing software, and QA shouldn't be done by the same people who are resetting and installing software partially because it's a different workflow and partially because it produces a sort of conflict-of-interest.

**We need a QA step because while it will take more time to get devices from intake to distribution, it's important to send people devices that do what they need in the first place.** If we send users shoddy devices that don't work, we're not doing our job and we're not really achieving our goal.  it will both save us time later on helping recipients who have received devices that don't work. As well, purely from a logistical standpoint, if we don't do QA we will end up spending more time helping users who have received devices that don't work as expected, dragging out the scope of the project to require much more intense tech support and dealing with much more returned devices than we would otherwise.

It would be good to work out easily followable guides for each thing a volunteer might have to do so that people can gain skills from working on this rather than having to have the skills in the first place.

**Comment by monotron** [comment]
Agreed, and I think documentation on our side is really important both for volunteers and for end-users. There will no doubt be common questions asked and problems we run into frequently, so I think it'd be good if we doubled down on documentation.

**Comment by cruxicheiros** [comment]
We may be able to allow groups to refine their own processes once they already exist if we take an approach where each stage is handled by a committee with sub-teams and a chair(s) who can handle admin for that group and help make decisions.

**Comment by cruxicheiros** [comment]
For example, the committee that handles wiping devices might have a sub-team for Windows devices. If that subteam works out that their process is ineffective or inefficient, they should be able to approach their chair with a suggestion for changing it.

**Comment by cruxicheiros** [comment]
Then it could be used, modified, or rejected based on whether it was a good idea or if it cut out important steps that were being taken for a specific reason.
